docs/glossary.md

```diff
@@ -21,7 +21,7 @@ NOTE:It has nothing to do with PostgreSQL pg_basebackup.
 
 ### Branch
 
-We can create branch at certain LSN using `zenith timeline branch` command.
+We can create branch at certain LSN using `neon timeline branch` command.
```

Comment:
```suggestion
We can create branch at certain LSN using `neon_local timeline branch` command.
```

The 'zenith' binary was renamed to 'neon_local' in commit ef40e404cf15cc335fbf6a226879e5358aa628eb.

Reply:
Thank you @hlinnaka, this makes sense. But I was curious, why is the binary called `neon_local` instead of `neon`?